README.md

```diff
@@ -45,11 +45,14 @@ FLAGS:
         --github-pre-lang    Use GitHub-style <pre lang> for code blocks
         --hardbreaks         Treat newlines as hard line breaks
     -h, --help               Prints help information
+        --no-config-file     Do not read config file
         --smart              Use smart punctuation
         --unsafe             Allow raw HTML and dangerous URLs
     -V, --version            Prints version information
 
 OPTIONS:
+    -c, --config-file <PATH>            Path to config file containing command-line arguments [default:
+                                        /Users/kivikakk/.config/comrak/config]
```

Comment:
Perhaps the documentation should show `~/.config/comrak/config`? Or is this dynamic?

Reply:
The output is dynamic, yeah — it'll show the actual default based on their XDG config path. The README excerpt is generated from running the actual binary on my machine, so it shows mine as a sample. :)